server/home/views.py

```diff
@@ -52,3 +52,25 @@ def search(request):
             })
 
     return render(request, 'home/search.html', {'query': query_text, 'results': results})
+
+
+# =================================================================================================
+
+
+def home(request):
+    return render(request, 'home/home.html')
+
+def navbar(request):
+    return render(request, 'navbar1.html')
```

Comment:
Why is the nav bar a separate route, can't it be imported in the template itself?